wagtail/admin/views/bulk_action.py

```diff
@@ -118,11 +123,14 @@ def post(self, request):
             before_hook_result = self.__run_before_hooks(self.action_type, request, objects)
             if before_hook_result is not None:
                 return before_hook_result
-            self.execute_action(objects)
+            self.execute_action(objects, **self.get_execution_context())
             after_hook_result = self.__run_after_hooks(self.action_type, request, objects)
             if after_hook_result is not None:
                 return after_hook_result
             success_message = self.get_success_message()
             if success_message is not None:
                 messages.success(request, success_message)
         return redirect(self.next_url)
+
+    def form_invalid(self, form):
+        return super().form_invalid(form)
```

Comment:
question: this does nothing (that `FormView` doesn't) , why is it required?

Reply:
Yes, I kept it there for some custom logic, then decided against it, and forgot to remove it.